.github/workflows/dev_pr/issue_check.js

```diff
@@ -78,11 +78,42 @@ async function commentNotStartedTicket(github, context, pullRequestNumber) {
     }
 }
 
+async function verifyGitHubIssue(github, context, pullRequestNumber, issueID) {
+    const issueInfo = await helpers.getGitHubInfo(github, context, issueID, pullRequestNumber);
+    if (issueInfo) {
+        if (!issueInfo.assignees.length) {
+            await github.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: pullRequestNumber,
+                body: ":warning: GitHub issue #" + issueID + " **has not been assigned in GitHub**, please assign the ticket."
+            })
+        }
+        if(!issueInfo.labels.length) {
+            await github.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: pullRequestNumber,
+                body: ":warning: GitHub issue #" + issueID + " **has no labels in GitHub**, please add labels for components."
+            })
+        }
+    } else {
+        await github.issues.createComment({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            issue_number: pullRequestNumber,
+            body: ":warning: GitHub issue #" + issueID + " could not be retrieved."
```

Comment:
This should be more of an error than a warning, but I guess the list of available emojis is limited.

Reply:
❌ ?